### PR TITLE
feat-five alerts

### PR DESCRIPTION
**Pull request by TheAmlanSaikia**
Description- Completed Alert  componenet.
Alerts

### CODE REVIEW

File: Alert.css [hunk 1]
```diff
@@ -0,0 +1,55 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
```
Comment (aniketsingh13) [inline Alert.css:5]: you can make reset CSS class for this

File: Alert.css [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  --primary-text-color: #490e60;
+  --secondary-color: #326771;
+  --secondary-text-color: #071203;
+  --success-color: #84f1c9;
+  --success-text-color: #1b7052;
+  --danger-color: #fcc5c5;
+  --danger-text-color: #dc3545;
+  --warning-color: #ffe7c5;
+  --warning-text-color: #ff9204;
+}
```
Comment (aniketsingh13) [inline Alert.css:18]: you can make utility class for this

**Review by aniketsingh13 — COMMENTED** [review]
Everything else look good for me and  consider this changes

File: Alert.css (hunk 2 above)
Comment (TheAmlanSaikia) [reply Alert.css:18]: Yes, after completeion of P0 components I will make utility class for all combined. Thanks

**Review by aniketsingh13 — APPROVED** [review]

**Review by Aditi0505 — APPROVED** [review]
Looks good to me

**Pull request merged**